python_examples/sim_speeds/old_basic_speeds.csv

```diff
@@ -0,0 +1,8 @@
+sim_time,pop_size
```

Comment:
Need to explain what these files are and where they came from. E.g., with a Readme file in the sim_speeds folder

Reply:
Added Readme file